specs/src/specs/resource_pricing.md

```diff
@@ -141,15 +141,15 @@ updated.
 
 ## Parameters
 
-There are three parameters that can be modified via governance to modify gas
+There are four parameters that can be modified via governance to modify gas
 usage.
 
-| Parameter       | Default Value | Description                             |
-|-----------------|---------------|-----------------------------------------|
-| consensus/max_gas     | -1     | The maximum gas allowed in a block. Default of -1 means this value is not capped.             |
-| auth/tx_size_cost_per_byte | 10 | Gas used per each byte used by the transaction. |
-| auth/sig_verify_cost_secp256k1 | 1000 | Gas used per verifying a secp256k1 signature |
-| blob/gas_per_blob_byte | 8 | Gas used per byte used by blob. Note that this value is applied to all encoding overhead, meaning things like the padding of the remaining share and namespace. See PFB gas estimation section for more details. |
+| Parameter       | Default Value | Description                             | Changeable via Governance |
+|-----------------|---------------|-----------------------------------------|---------------------------|
+| consensus/max_gas     | -1     | The maximum gas allowed in a block. Default of -1 means this value is not capped.             | True                      |
```

Comment:
[question][not blocking] why is this `conesus/max_gas` here and `consensus.block.MaxGas` [here](https://github.com/celestiaorg/celestia-app/pull/2973/files#diff-3b7c00e0a4ce251ab33bcf179793c288a0affef839a4b1700a8cec00a202ad2aR31).

[question] Is there a test that verifies `max_gas` is modifiable in https://github.com/celestiaorg/celestia-app/pull/2973?

Reply:
I assumed it was referring to `consensus.block.MaxGas` because of the description "The maximum gas allowed in a block.".

For the second question, yes it is in [line 182](https://github.com/celestiaorg/celestia-app/blob/abec39b1cdc04f76373959fc87000401d82f5b83/x/paramfilter/test/governance_params_test.go#L182).

Reply:
Thanks can resolve.